### ISSUE TITLE
clip.py部分变量没有定义，报红

### ISSUE THREAD

**Issue by zhounan-n** [Tarasom123/ERIC-FND#10]
__init__方法中

self.img_pos_drop = nn.Dropout(p=dropout)

self.text_embed.weight = nn.Parameter(torch.from_numpy(W))
        self.text_encoder = nn.Sequential(nn.Linear(d_text, d_model),
                                          nn.LayerNorm(d_model),
                                          TextPositionEmbed(seq_len, d_model, dropout))
        s = seq_len // 2 + 1

dropout，W，d_text，d_model，seq_len 这几个变量报红，辛苦再检查一下

**Comment by zhounan-n** [comment]
还有这个TextPositionEmbed
